index.html

```diff
@@ -10,14 +10,22 @@
 <body>
     <header>
         <a href="https://github.com/Elisha-Tetteyfio/Portfolio-setup-mobile-version-skeleton" class="menu-element" id="name">Elisha</a>
-        <nav>
+        <nav class="mobile">
             <img src="Icons/Union.svg" alt="Menu" class="menu-element" id="menu">
         </nav>
+        <div class="desktop">
+            <ul>
+                <li> <a href="#Portfolio">Portfolio</a> </li>
+                <li> <a href="#About">#About</a> </li>
+                <li> <a href="#Contact">Contact</a> </li>
+                <li><img src="Icons/Mail.svg" alt="mail"></li>
+            </ul>
+        </div>
     </header>
     <section id="headline">
         <h1 id="headline-1">Elisha Tetteyfio</h1>
         <h2 id="headline-2">Software developer</h2>
-        <p id="content"> <span>I</span> can help you build a product, feature or website. Look through some of my works and experiences! If you like what you see and have a project you need coded,  don't hesitate to contact me.</p>
+        <p id="content">I can help you build a product, feature or website. Look through some of my works and experiences! If you like what you see and have a project you need coded,  don't hesitate to contact me.</p>
     </section>
     <hr>   
     <section id="skillset">
```

Comment:
Figma design
![Screenshot from 2022-04-30 21-17-23](https://user-images.githubusercontent.com/77842237/166117815-b1432dbb-f9cc-4c4b-8d25-e9c8434ca99e.png)

Your Design
![Screenshot from 2022-04-30 21-17-06](https://user-images.githubusercontent.com/77842237/166117825-5e784ab2-7d4b-4b90-8c15-e095cc4ecd13.png)


- [ ] Please the line should not cover the whole page. if not it should just contain the skills and the white ball should be on the right like in Figma design.